### DISCUSSION TITLE
知识区位置改变

### DISCUSSION THREAD

**Discussion by Auroradanica** [the1812/Bilibili-Evolved#2046] [提问 / Q&A] [answered]
哔哩哔哩知识区的地址发生了变化，希望尽快解决，感谢

**Comment by Geeyun-JY3** [comment, marked as answer]
重复：https://github.com/the1812/Bilibili-Evolved/discussions/1511#discussioncomment-811558
预览版早修了：[[预览]v1.12.11](https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.11-preview)
